RecoHGCal/TICL/python/ticl_iterations.py

```diff
@@ -22,6 +22,39 @@ def TICL_iterations_withReco(process):
 
   process.TICLLayerTileProducer = ticlLayerTileProducer.clone()
 
+  process.SeedingTrk = seedingRegionProducer.clone(
```

Comment:
module instances should start with lower case (this comment applies to everything in this file)

Reply:
ok done